solr/solr-ref-guide/modules/configuration-guide/pages/caches-warming.adoc

```diff
@@ -343,6 +355,34 @@ A key best practice is to modify these defaults before taking your application t
 There is no point in auto-warming your Searcher with the query string "static firstSearcher warming in solrconfig.xml" if that is not relevant to your search application.
 ====
 
+### Multi-threaded listener warming
+
+If you have a large list of queries you are running via the listeners you might want to enable multi-threading listener warming by specifying the `<threads>` element:
+
+[source,xml]
+----
+<listener event="newSearcher" class="solr.QuerySenderListener">
+  <threads>4</threads>
```

Comment:
Normally we use `<int name="threads">4</int>` style of namedList config many places in solrconfig. I suppose you get some type casting for free then? Personally I  am not opposed to a more custom XML syntax, just asking whether you considered that format as an alternative?

Reply:
You're absolutely right. That was a documentation typo on my part. It should be `<int name="threads">4</int>` which is that is what my local configuration files for testing are using. I've updated the PR.